### PR TITLE
TC_01.001.05_A | Add description to the new project

### PR DESCRIPTION
**Pull request by Russavell**
TC_01.001.05_A | Add description to the new project

https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/243

### CODE REVIEW

File: cypress/e2e/freestyleProjectAddDescription.cy.js [hunk 1]
```diff
@@ -50,4 +50,13 @@ describe("US_01.001 | FreestyleProject > Add description", () => {
       .should("be.visible")
       .and("have.text", newDescription);
   });
+
+  it("TC_01.001.05_A | Add description to the new project", () => {
+    cy.get('[name="description"]').type(description);
+    cy.get('[name="Submit"]').click();
+
+    cy.get('[class="jenkins-app-bar__content jenkins-build-caption"]').should('have.text', newItemName);
+    cy.get('[id="description"]').should('be.visible').and('have.text', description);
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectAddDescription.cy.js:59]: could use the shorthand '#description' instead '[id="description"]'

**Comment by adiuzhikov** [comment]
It appears that there is an error in the test case steps. Please revise them to align with the auto test. You are testing the Freestyle Project -> Add Description feature, but the test case steps are testing a different feature, List View.

File: cypress/e2e/freestyleProjectAddDescription.cy.js (hunk 1 above)
Comment (Russavell) [reply cypress/e2e/freestyleProjectAddDescription.cy.js]: Is the rest looks good? Please review again when you have a chance.

File: cypress/e2e/freestyleProjectAddDescription.cy.js [hunk 2]
```diff
@@ -50,4 +50,13 @@ describe("US_01.001 | FreestyleProject > Add description", () => {
       .should("be.visible")
       .and("have.text", newDescription);
   });
+
+  it("TC_01.001.05_A | Add description to the new project", () => {
+    cy.get('[name="description"]').type(description);
+    cy.get('[name="Submit"]').click();
+
+    cy.get('[class="jenkins-app-bar__content jenkins-build-caption"]').should('have.text', newItemName);
```
Comment (Stavolg) [inline cypress/e2e/freestyleProjectAddDescription.cy.js:58]: pls check the selector, maybe you could use 'h1.page-headline' or 'h1.job-index-headline'

File: cypress/e2e/freestyleProjectAddDescription.cy.js (hunk 1 above)
Comment (Stavolg) [reply cypress/e2e/freestyleProjectAddDescription.cy.js]: looks good!

File: cypress/e2e/freestyleProjectAddDescription.cy.js (hunk 2 above)
Comment (Russavell) [reply cypress/e2e/freestyleProjectAddDescription.cy.js:58]: What is wrong with the selector? It did not pass for you?

Comment (Stavolg) [reply cypress/e2e/freestyleProjectAddDescription.cy.js:58]: I don't think it's wrong, but it's hard to read code

**Comment by Russavell** [comment]
> It appears that there is an error in the test case steps. Please revise them to align with the auto-test. You are testing the Freestyle Project -> Add Description feature, but the test case steps are testing a different feature, List View.

My steps are almost identical to TC-1. TC-1 is approved.

**Review by Stavolg — APPROVED** [review]

**Review by antonbloha — APPROVED** [review]

**Comment by Russavell** [comment]
> > My steps are almost identical to TC-1. TC-1 is approved.
> 
> Then can you explain why you are not following your own steps from your Test Case?
> 
> > Click "+" to add a new project
> > Select Type "List view"
> 
> I cannot see anywhere in your test you are doing that.

Oh. I got you. Let me fix it. You are right.

**Comment by Russavell** [comment]
> > > My steps are almost identical to TC-1. TC-1 is approved.
> > 
> > 
> > Then can you explain why you are not following your own steps from your Test Case?
> > > Click "+" to add a new project
> > > Select Type "List view"
> > 
> > 
> > I cannot see anywhere in your test you are doing that.
> 
> Oh. I got you. Let me fix it. You are right.

TC re-wrote. Thank you for your feedback.

**Pull request merged**